dev-util/umockdev/umockdev-0.10.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit python-any-r1
+
+MY_PV=${PV%??}
+
+DESCRIPTION="Mock hardware devices for creating unit tests"
+HOMEPAGE="https://github.com/martinpitt/umockdev/"
+SRC_URI="https://launchpad.net/${PN}/trunk/${PV}/+download/${PN}-${MY_PV}.tar.xz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs test"
+
+RDEPEND="virtual/libgudev:=
+	virtual/libudev:=
+	>=dev-libs/glib-2.32:2"
+DEPEND="${RDEPEND}
+	test? ( ${PYTHON_DEPS} )
+	app-arch/xz-utils
+	virtual/pkgconfig"
+
+RESTRICT="test"
```

Comment:
Why? This needs an explanatory comment.

Reply:
I mainly used the old 0.9 ebuild to bump then checked if it worked. I think it was restricted because some tests are known to crash stuff, maybe that got fixed or maybe that was not the reason. I'll dig deeper in the next days and come up with a new ebuild.